code/restful-api/restful-api/Models/Possui.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RestfulApi.Models
+{
+    public class Possui
```

Comment:
Same for above, if we go with that naming approach, this should be `UsuarioCartaoDeCredito`.

Reply:
Feito